theme/README.md

```diff
@@ -49,6 +49,16 @@ To go into the theme workbench interface, sign into _Zendesk Guid_ as an admin,
 
 ### Import from Github
 
+#### Sandbox Theme
+
+-   On the `sandbox` branch, pull the latest changes from `master` and push to origin. It will trigger a github action that will build the theme and push the output to the `sandbox-deployed` branch.
```

Comment:
Can we update to the following to keep the language consistent?

```On the `sandbox` branch, pull the latest changes from the `master` branch and push the branch to the `origin` repository. It will trigger a github action that will build the theme and push the output to the `sandbox-deployed` branch.```

Reply:
Rephrased at dcf50e1, what do you think?

Reply:
That looks good.  Thanks @rdai10!